TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.m

```diff
@@ -66,14 +66,14 @@
 #import "TSDKRootLinks.h"
 #import "TSDKApplePaidFeature.h"
 
-static NSMutableArray *supportedSDKObjects;
+static NSArray *supportedSDKObjects;
 static NSArray *knownCompletionTypes;
 
 @implementation TSDKObjectsRequest
 
 + (NSArray *)supportedSDKObjects {
     if (!supportedSDKObjects) {
-        NSMutableArray *supporteObjects = [[NSMutableArray alloc] init];
+        NSMutableArray *supportedObjects = [[NSMutableArray alloc] init];
```

Comment:
If this is a thread safety issue, then this method is still not thread safe. Perhaps wrap the writes in synchronize statement?